lms/templates/register-form.html

```diff
@@ -240,16 +239,12 @@ <h2 class="sr">${_("Account Acknowledgements")}</h2>
       % endif
 
       % if settings.REGISTRATION_EXTRA_FIELDS['honor_code'] != 'hidden':
-      ## If the stanford theme isn't enabled, check if we have an Honor Code link in our marketing map
-      % if not stanford_theme_enabled() and marketing_link('HONOR') and marketing_link('HONOR') != '#':
+      ## check if we have an Honor Code link in our marketing map
+      % if marketing_link('HONOR') and marketing_link('HONOR') != '#':
       <div class="field ${settings.REGISTRATION_EXTRA_FIELDS['honor_code']} checkbox" id="field-honorcode">
```

Comment:
I think block should be removed here since it is only visible in standford theme.